README.md

```diff
@@ -148,6 +149,7 @@ absence of a ground truth.
 - [TraJClassifier](https://imagej.net/plugins/trajclassifier) - Fiji plugin that loads trajectories from TrackMate, characterizes them using TraJ and classifiies them into normal diffusion, subdiffusion, confined diffusion and directed/active motion by a random forest approach (through Renjin).
 - [QuimP](https://github.com/CellDynamics/QuimP) - Software for tracking cellular shape changes and dynamic distributions of fluorescent reporters at the cell membrane.
 - [Usiigaci](https://github.com/oist/usiigaci) - Stain-free cell tracking in phase contrast microscopy enabled by supervised machine learning.
+- [Trackpy](https://soft-matter.github.io/trackpy) - Trackpy: Fast, Flexible Particle-Tracking Toolkit (Python).
```

Comment:
From the description it seems that this package belongs to the "Cell migration and particle tracking" section.

Reply:
Yes it is in the same section.